src/Codec/Xlsx/Types.hs

```diff
@@ -28,29 +29,37 @@ import           Data.Map (Map)
 import qualified Data.Map as M
 import           Data.Text (Text)
 import qualified Data.Text as T
-
+import           Text.XML (Name)
 
 -- | Cell values include text, numbers and booleans,
 -- standard includes date format also but actually dates
 -- are represented by numbers with a date format assigned
 -- to a cell containing it
-data CellValue = CellText   Text
-               | CellDouble Double
-               | CellBool   Bool
+data CellValue = CellText    Text
+               | CellDouble  Double
+               | CellBool    Bool
                deriving (Eq, Show)
 
--- | Currently cell details include only cell values and style ids
--- (e.g. formulas from @\<f\>@ and inline strings from @\<is\>@
--- subelements are ignored)
+data CellFormula = CellFormula { _cellFormulaValue :: Maybe Text
+                               , _cellFormulaAttrs :: [(Name, Text)] 
```

Comment:
But what's the reason? The only attribute in spec which is not from Spreadsheet ML is `xml:space` so I'd use either extra record field for it and [(Text,Text)] for other attributes or fields for every one of them (spec shows me only 12 them :) )